python/pyfury/_serialization.pyx

```diff
@@ -1058,6 +1061,11 @@ cdef class Fury:
             self.unpickler = Unpickler(buffer)
         if unsupported_objects is not None:
             self._unsupported_objects = iter(unsupported_objects)
+        if self.language == Language.XLANG:
+            magic_numer = buffer.read_int16()
+            assert magic_numer == MAGIC_NUMBER, (
+                "The XLANG cross-language protocol is not currently used. For details, see "
```

Comment:
Clang and cross language seems a little redundant. Other parts look good to me. But could we refine the error message？

Reply:
How about the error msg below?
`The xlang serialization specification must start with 0x62d4. Please confirm whether the serialization is based on the xlang protocol. For details, see https://github.com/apache/incubator-fury/blob/main/docs/specification/xlang_serialization_spec.md`

Reply:
how about `The fury xlang serialization must start with magic number 0x62d4. Please check whether the serialization is based on the xlang protocol and the data didn't corrupt.`

Reply:
Great! updated.